frontend/src/api/browserTiles.js

```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 const fetchBrowserTiles = async ({ model, version }) => {
   const params = { model, version };
   const { data } = await axios.get('/random-components', { params });
-  return data;
+  return { model, version, ...data };
```

Comment:
There was a need to check that the tiles in the store are the same as the selected model. In the case that tiles would be loaded, and the model is changed, new tiles would have to be loaded for this newly selected model.

Reply:
Can you point me to the specific point where this is used? I’m wondering how this interaction can happen.

Reply:
Hmm it's not being used actually. I suspect I have forgotten mid-way what I was working on. The place for it would be under `getTilesData` in `GemBrowser.vue`, and the tiles should never be reloaded when revisiting this page - only when people press the `random` button.

Reply:
Sorry I must not have been looking closely enough - [here](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/refactor/explorer-pages/frontend/src/components/explorer/GemBrowser.vue#L130-L134) is why the model and version are needed in the tiles.

Reply:
I took a look at this and suggest one of two alternatives:

1. In `App.vue`, is the `<keep-alive>` wrapper around `<router-view>` still needed? If not we can completed remove the `beforeUpdate` block in `GemBrowser.vue`, as it would be re-mounted every time. We might need to modify the logic for determining if something is loading. I think `keep-alive` was only needed before for tying together all the "child" pages of gem browser but you have isolated them now. So this would be the best option if possible.
2. If we still need the `keep-alive` for some reason we could add a (deep) watch on the store's `model` instead and compare the name and version for the before and after models.

Reply:
I'll go for 1 and also see if the tiles should be cleared after a model selection.